src/app/static/src/tests/components/stepper.test.ts

```diff
@@ -279,20 +295,20 @@ describe("Stepper component", () => {
         expect([1, 2, 3, 4, 5].filter(i => steps.at(i).props().enabled).length).toBe(0);
     });
 
-    it("upload surveys step is enabled when baseline step is complete", () => {
+    it("Review inputs step is enabled when baseline step is complete", () => {
```

Comment:
Should really have two tests to test both that Review inputs is *not* enabled when we have only completedBaselineState, and also that it is enabled  and complete (and subsequent step is enabled) when we have both completedBaselineState and completeSurveyAndProgram state. And rename each test accordingly.